src/service/url-replacements-impl.js

```diff
@@ -451,7 +451,7 @@ export class UrlReplacements {
       return Promise.resolve();
     } else if (metric < 0) {
       // Metric is not yet available. Retry after a delay.
-      return loadPromise(this.win_).then(() => {
+      return loadPromise(this.win_.document).then(() => {
```

Comment:
Alternatively, if we want the proper `window.onload`-like behavior, I believe, we need to listen on doc's `readyState == complete`. This is not what `onDocumentReady` does now, but we can easily add that callback as well. We can put it in the same `document-ready.js` and call it `onDocumentComplete` or something like that. From what I can see this is the closest thing to `window.onload`. So, let's pick the right event and move with that.